.travis.yml

```diff
@@ -16,6 +16,18 @@ matrix:
   fast_finish: true
   include:
   - rvm: 2.1.9
+    bundler_args: --without development
+    dist: trusty
+    env: PUPPET_INSTALL_TYPE=agent BEAKER_debug=true BEAKER_set=docker/ubuntu-14.04 CHECK=beaker
+    services: docker
+    sudo: required
+  - rvm: 2.1.9
+    bundler_args: --without development
+    dist: trusty
+    env: PUPPET_INSTALL_TYPE=agent BEAKER_debug=true BEAKER_set=docker/centos-7 CHECK=beaker
+    services: docker
+    sudo: required
+  - rvm: 2.1.9
```

Comment:
Can we use 2.4.1 here?

Reply:
I think this would need to come from modulesync somehow?
https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/.travis.yml.erb#L39

Reply:
Now that it's updated in modulesync, I'll give it a shot as an extra test.